src/Protocol/Diaspora.php

```diff
@@ -4006,12 +4007,12 @@ private static function createProfileData(int $uid): array
 	/**
 	 * Sends profile data
 	 *
-	 * @param int  $uid    The user id
-	 * @param bool $recips optional, default false
+	 * @param int   $uid    The user id
+	 * @param array $recips optional, default empty array
 	 * @return void
 	 * @throws \Exception
 	 */
-	public static function sendProfile(int $uid, bool $recips = false)
+	public static function sendProfile(int $uid, array $recips = [])
```

Comment:
Have you checked all occurrences?

Reply:
I had a look into the function that clearly uses an array. Also my IDE threw a lot of warnings at me, when I opened that file.